cpp/src/backends/process/model_worker.cc

```diff
@@ -15,56 +15,57 @@ namespace torchserve {
     if (socket_type == "unix") {
       socket_family = AF_UNIX;
       if (socket_name.empty()) {
-        LOG(FATAL) << "Wrong arguments passed. No socket name given.";
+        TS_LOG(FATAL, "Wrong arguments passed. No socket name given.");
       }
-      
-      std::filesystem::path s_name_path(socket_name);
-      if (std::remove(socket_name.c_str()) != 0 && std::filesystem::exists(s_name_path)) {
-        LOG(FATAL) << "socket already in use: " << socket_name;
+
+      std::experimental::filesystem::path s_name_path(socket_name);
+      if (std::remove(socket_name.c_str()) != 0 && std::experimental::filesystem::exists(s_name_path)) {
```

Comment:
Not sure if upgrading gcc is the best idea since gcc-9 is not the default on the current ubuntu LTS. We can use a [macro](https://stackoverflow.com/questions/53365538/how-to-determine-whether-to-use-filesystem-or-experimental-filesystem) maybe

Reply:
gcc-9 is must. it is default in ubuntu-20.